tcms/testruns/views.py

```diff
@@ -1193,7 +1187,7 @@ def env_value(request):
     test_runs = TestRun.objects.filter(run_id__in=request.GET.getlist('run_id'))
 
     class RunEnvActions(object):
-        def __init__(self, requet, test_runs):
+        def __init__(self, request, test_runs):
```

Comment:
Actually, IMO this has been working fine despite the typo. In the `__init__` of `RunEnvActions` we instantiate `self.request = requests`, however `self.request` is not used anywhere in the class. Instead, what is used is `request` and since it is an inner class I suppose that it can see the `request` attribute of `env_value` and use it without any problems. As far as I can see there are good tests for each scenario, so the fact that they are passing means everything is fine. The next step may be extracting `RunEnvActions` and testing it separately as it was done for some of the other inner classes we encountered. A part of this step would be replacing `request` with `self.request`, but for now everything is working as expected and IMO things can be left as they are.

Reply:
I think you are correct in this interpretation. Just open an issue for the extraction and refactoring of this class.

Reply:
#319